modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/TextStyles.kt

```diff
@@ -215,13 +215,14 @@ fun TextH70(
     modifier: Modifier = Modifier,
     textAlign: TextAlign? = null,
     color: Color = MaterialTheme.theme.colors.primaryText01,
+    fontWeight: FontWeight? = null,
```

Comment:
```suggestion
    fontWeight: FontWeight = FontWeight(500),
```
The other option would be to set the default value here, similar to the value above.